about.html

```diff
@@ -96,47 +96,7 @@ <h1>Welcome to  ChessMate chess club(CCC) </h1>
     into top spot in South Africa.</p>
 </div>
 
-<h2 style="text-align:center">Our Team</h2>
-<div class="row">
-  <div class="column">
-    <div class="card">
-      <img src="./image/sandro-g.jpg" alt="Jane" style="width:35%">
-      <div class="container">
-        <h2>Andrea Rasp</h2>
-        <p class="title">Founder</p>
-        <p class="P-text">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Vel aliquam harum voluptates suscipit! Debitis magnam distinctio vero repudiandae, molestias quam.</p>
-        <p class="email">Andreaj@chessMate.com</p>
-        <p><button class="button">Contact</button></p>
-      </div>
-    </div>
-  </div>
-
-  <div class="column">
-    <div class="card">
-      <img src="./image/jack.jpg" alt="Mike" style="width:35%">
-      <div class="container">
-        <h2>Jack Ross</h2>
-        <p class="title"> Director</p>
-        <p class="P-text">Lorem ipsum dolor sit amet consectetur adipisicing elit. Sint quo sed excepturi tenetur temporibus quas quidem, fugit at atque ullam deserunt hic rem.</p>
-        <p class="email">jackr@chessMate.com</p>
-        <p><button class="button">Contact</button></p>
-      </div>
-    </div>
-  </div>
-
-  <div class="column">
-    <div class="card">
-      <img src="/image/tide_trasher.jpg" alt="John" style="width:35%">
-      <div class="container">
-        <h2>John Tide</h2>
-        <p class="title">Designer</p>
-        <p class="P-text" >Lorem ipsum dolor sit, amet consectetur adipisicing elit. Perferendis ad fugiat odio quo voluptate eaque autem corrupti ipsum sapiente, consequatur molestiae, deserunt quidem ipsa.</p>
-        <p class="email">johnj@chessMate.com</p>
-        <p><button class="button">Contact</button></p>
-      </div>
-    </div>
-  </div>
```

Comment:
- As requested by the previous reviewer, kindly add the footer part of your project as seen on the about page below :point_down: 
![image](https://user-images.githubusercontent.com/97834160/233808346-600ca4a5-5c82-4019-910c-c87d79a62e18.png)